sourcemash/frontend/frontend.py

```diff
@@ -1,9 +1,13 @@
 from . import bp
 
-from flask import render_template
+from flask import render_template, url_for
 
 
 @bp.route('/')
 @bp.route('/index')
 def index():
     return render_template('index.html')
+
+@bp.route('/login')
```

Comment:
It's not. It makes a post to /login which is Flask-Security, but the way it processes the HTML submit is that is adds the variables to the URL which we don't want since we're not allowing the page to refresh until the login credentials are correct.